packages/clay-datepicker/src/Button.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from './Icon';
+
+/**
+ * @param {Object} param
+ * @return {React.createElement}
+ */
+function Button({
+	className,
+	icon,
+	monospaced,
+	onClick,
+	size,
+	spritemap,
+	style,
+	type,
+}) {
+	const classNames = classnames('btn', {
+		'btn-monospaced': monospaced,
+		[`btn-${size}`]: size,
+		[`btn-${style}`]: style,
+		[className]: className,
+	});
+
+	return (
+		<button className={classNames} onClick={onClick} type={type}>
+			<Icon spritemap={spritemap} symbol={icon} />
+		</button>
+	);
+}
+
+Button.propTypes = {
+	className: PropTypes.string,
+	icon: PropTypes.string,
+	monospaced: PropTypes.bool,
+	onClick: PropTypes.func,
+	size: PropTypes.oneOf(['sm']),
+	spritemap: PropTypes.string,
+	style: PropTypes.oneOfType([
+		PropTypes.bool,
+		PropTypes.oneOf([
+			'link',
+			'primary',
+			'secondary',
+			'unstyled',
+			'primary',
+		]),
+	]),
+	type: PropTypes.oneOf(['button', 'reset', 'submit']),
+};
+
+Button.defaultProps = {
+	monospaced: false,
+	style: 'primary',
+	type: 'button',
+};
+
+export {Button};
+export default Button;
```

Comment:
What's the rationale for exporting twice here?

Reply:
hey @wincent, Actually we ended up leaving this as standard in having both, this ended up staying in our `generator-metal-clay`.

Reply:
I created https://github.com/liferay/clay/pull/1453 so that we have a place to discuss this further.

Reply:
Great!!